### ISSUE TITLE
Unable to resolve import

### ISSUE THREAD

**Issue by johnnyzen** [square/wire#1311]
HI all

Following on from your dinosaur example....

I am getting (using: api 'com.squareup.wire:wire-runtime:3.0.2')

Syntax: proto3

`Execution failed for task ':app:generateProtos'.
> unable to resolve 1 imports:
    testing/tastes/tastes.proto
  searching 0 proto paths:`

My understanding is that I do not need to define anything in gradle for looking for imports?
- But when it says "searching 0 proto paths", it makes me wonder....

My gradle is:

`wire {
    kotlin {
        android = true
    }
}`

File Structure is

```
main
      -> proto
           -> testing
                -> people
                     -> person.proto
                -> tastes
                     -> tastes.proto
```

my import line in person.proto is: `import "testing/tastes/tastes.proto";`

I would be very grateful for any assistance :)

**Comment by Egorand** [comment]
Is it `person.proto` that's trying to import `tastes.proto`? Is there a `package testing.tastes;` in `tastes.proto`. Also FYI, Wire currently doesn't support proto3.

**Comment by johnnyzen** [comment]
Yes @Egorand yes, person.proto needs tastes.proto.
package for person is `testing.people;`
package for tastes is 'testing.tastes'

**Comment by Egorand** [comment]
I'm going to close this ticket - we use the issue tracker for bug reports and feature requests, not for usage questions. Please move your question to [StackOverflow](https://stackoverflow.com/questions/tagged/square-wire) and provide more detail - if you're able to share the actual protos that'd be great. Will try my best to help.

**Comment by johnnyzen** [comment]
Hmmm maybe no support for proto3 is the reason, I will try with proto2 - see if I get different result

**Comment by johnnyzen** [comment]
Thanks @Egorand  I have posted here https://stackoverflow.com/questions/59290722/unable-to-resolve-x-imports if anyone happens to come across this closed ticket with same problem.

Best wishes

**Comment by johnnyzen** [comment]
An update. I suspect an issue with windows path (I am using windows). I can get the imports to work if I place the tastes.proto file in the src/main/proto folder instead of src/main/proto/testing/tastes folder. I have tried adding the tastes directory to protoPath in gradle options, but all I get when I do this is "Collection contains no element matching the predicat" error. BTW using protoPath with a empty folder location seems the only way to compile (still get unable to resolve 1 imports issue though :( )but then get Cannot find an example of how proto needs be structured when using protoPath 

I think there is an issue here with the way the plugin handles paths, but I need some help.. I noticed in the code there is some debug code to list the paths... can I enable this ? @Egorand
